pages/products.css

```diff
@@ -0,0 +1,77 @@
+@import url('/style.css');
+
+.filter-value {
+    display: flex;
+    flex-direction: column;
+    margin: 1rem;
+}
+
+.filters {
+    display: flex;
+    flex-direction: column;
+    padding: 1rem;
+}
+
+.filter {
+    accent-color: var(--cta);
+}
+
+.filter-wrapper>h3 {
+    margin: 0;
+}
+
+.vertical-card {
+    display: flex;
+    flex-direction: column;
+    max-width: 14rem;
+    border: 2px var(--border);
+    box-shadow:
+        2px 4px 8px hsl(0deg 0% 0% / 0.25);
```

Comment:
Suggestion: You can use a CSS variable for `box-shadow` as you're using it at multiple places.

Reply:
Thanks for pointing it out i have made the changes and will merge the pr